src/util/tubemap.js

```diff
@@ -3671,8 +3688,12 @@ function drawTrackRectangles(rectangles, type, groupTrack) {
 }
 
 function compareCurvesByLineChanges(a, b) {
```

Comment:
This no longer compares by the number of vertical lanes changed, so I think it probably needs a different name now,